pkg/tests/end_to_end_tests/create_test.go

```diff
@@ -1539,10 +1539,16 @@ func TestConfigMaintenanceJobs(t *testing.T) {
 			t.Fatal("Incorrect number of jobs at startup")
 		}
 
-		changeJobs := func(numJobs int, scheduleInterval time.Duration) {
-			_, err = db.Exec(context.Background(), "SELECT config_maintenance_jobs($1, $2)", numJobs, scheduleInterval)
+		changeJobs := func(numJobs int, scheduleInterval time.Duration, config *string, configErr bool) {
+			_, err = db.Exec(context.Background(), "SELECT config_maintenance_jobs($1, $2, $3)", numJobs, scheduleInterval, config)
 			if err != nil {
-				t.Fatal(err)
+				if !configErr {
+					t.Fatal(err)
+				}
+				return
```

Comment:
If I am not wrong, t.Fatal() stops the test by itself, so return right be trivial.

Reply:
this is need if configErr is true